AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/AWSModelReconciliationQueue.swift

```diff
@@ -52,6 +52,11 @@ final class AWSModelReconciliationQueue: ModelReconciliationQueue {
 
     weak var storageAdapter: StorageEngineAdapter?
 
+    internal var isFullSync: Bool
```

Comment:
1. As noted above, the reconciliation queue should not be tracking the type of sync being used 
1. `internal` is the default access level, no need to specify it